scripts/README.md

```diff
@@ -44,12 +44,17 @@ To use this mode please add parameter:
 
 #### List of available options
 ##### login
+  - Additional parameters: --proxy_host, --proxy_port
   - Additional parameters provided with environment variables: certificate path with 
       MANAGEMENT_CA_CERT_PATH
   - Usage example:
     ```
     ./imm -a mgmt.example.com login
     ```
+  - Using proxy and offline login option:
+    ```
+    ./imm -o login --proxy_port 911 --proxy_host example.proxy.com
```

Comment:
At some point we could think about single parameter with host:port format (--proxy example.proxy.com:911). 
It would require additional logic in authenticating script, but might be more convenient for users.

Reply:
@pawel-pieniazek :arrow_up: :arrow_up: :arrow_up: